portfolio/src/main/webapp/script.js

```diff
@@ -39,27 +37,59 @@ function addHobby() {
  */
 function addProjectDescription(project) {
   
-  const details = {
-    nlp: 'Building language models',
-    ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
-    helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
-    unknown: 'Something is wrong, project doesn\'t exist'
-  };
-  
-  const colors = {
-    nlp: '#6ccfe0',
-    ta: '#f7d36f',
-    helmet: '#e0b9f0',
-    unknown: 'black'
-  };
+  const projectName = project.toUpperCase(); 
+
+  const projectNameToData = {
+    NLP: createProjectData('Building language models', '#6ccfe0'),
+    TA: createProjectData('Held weekly office hours and review sessions for introductory Data Structure and Algorithms class', '#f7d36f'),
```

Comment:
This line and the one below are a bit long. We can separate into multiple lines.

ex:

  TA: createProjectData(
    '...',
    '...'),

Reply:
Done.